examples/datasource-basic/package.json

```diff
@@ -5,14 +5,14 @@
   "scripts": {
     "build": "webpack -c ./.webpack/webpack.prod.conf.js",
     "dev": "webpack -w -c ./.webpack/webpack.dev.conf.js",
-    "e2e": "TZ=UTC grafana-e2e run",
-    "e2e:open": "TZ=UTC grafana-e2e open",
-    "e2e:update": "TZ=UTC grafana-e2e run --update-screenshots",
+    "e2e": "docker run --rm --add-host=host.docker.internal:host-gateway --volume $(pwd):/e2e --workdir /e2e --env='CYPRESS_HOST=host.docker.internal' --env='CYPRESS_BASE_URL=http://host.docker.internal:3000' --entrypoint '/bin/bash' cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97 -c './node_modules/.bin/cypress install && yarn grafana-e2e run'",
+    "e2e:update": "docker run --rm --add-host=host.docker.internal:host-gateway --volume $(pwd):/e2e --workdir /e2e --env='CYPRESS_HOST=host.docker.internal' --env='CYPRESS_BASE_URL=http://host.docker.internal:3000' --entrypoint '/bin/bash' cypress/included:9.5.1-node16.14.0-slim-chrome99-ff97 -c './node_modules/.bin/cypress install && yarn grafana-e2e run --update-screenshots'",
     "test": "jest --watch --onlyChanged",
     "test:ci": "jest --maxWorkers 4",
     "typecheck": "tsc --noEmit",
     "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
-    "lint:fix": "yarn lint --fix"
+    "lint:fix": "yarn lint --fix",
+    "serve": "GRAFANA_VERSION=8.3.6 docker-compose up -d --build"
```

Comment:
**Note for us:** I think we will need to make sure that update the `GRAFANA_VERSION` here as well with the update command.

Reply:
Sounds like a good idea!